packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -254,6 +261,18 @@ export default class Agent extends EventEmitter<{|
     }
   };
 
+  getDataForCopy = ({id, property, rendererID}: GetDataForCopyParams) => {
+    const renderer = this._rendererInterfaces[rendererID];
+    if (renderer == null) {
+      console.warn(`Invalid renderer id "${rendererID}" for element "${id}"`);
```

Comment:
Tiny change request : line 267, `Invalid renderer id "${rendererID}" for element "${id}"` is repeated about 10 times in this file, is if worth it if we extract this to a meaningful const at the top of the file? 🙃  

This approach will eventually bring in mind issues (in a separate PR / NFR in the backlog ) such as:
- If we are repeating texts across a file, are those texts being used across other files?
- If so, how about centralising texts?
- Is centralising the texts worth it (i18n etc)? Will those texts eventually need translation as react grows to an international audience (eg: China)?

Reply:
Sounds sensible to me.  Have updated the PR